core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestPruneSpatialJoinChildrenColumns.java

```diff
@@ -45,7 +69,10 @@ public void testPruneOneChild()
                             p.values(a, unused),
                             p.values(b, r),
                             ImmutableList.of(a, b, r),
-                            expression("ST_Distance(a, b) <= r"));
```

Comment:
Can we handle this by adding a `rewriteFunctionCall` to `io.trino.sql.ExpressionUtils#rewriteIdentifiersToSymbolReferences`?